### ISSUE TITLE
Reader: Instagram image missing in post content

### ISSUE THREAD

**Issue by bluefuton** [Automattic/wp-calypso#25399]
#### Steps to reproduce
1. Starting at URL: http://calypso.localhost:3000/read/feeds/9092981/posts/1284960282

After "Have you read my Guide to Italian aperitivo?", there's an embedded image that's being stripped. We suspect the `makeImagesSafe` normalization rule is the culprit, possibly because the image URL can't be run through Photon: 

http://instagr.am/p/BGzZqxOHrhQ/media/?size=l

Raised in p2-p5PDj3-2jx.

**Comment by roo2** [comment]
I couldn't reproduce this. It looks like the blog and instagram link are both no longer available.
I set up a dummy post with an instagram embeded image and was able to see it as expected 


I looked through p5PDj3-2jx-p2 but couldn't work out what other conditions needed to be met to repro this.

<img width="1077" alt="Screen Shot 2023-02-06 at 3 33 26 pm" src="https://user-images.githubusercontent.com/22446385/216891488-3b0d0c90-7e8a-4ddb-895a-831185c6613e.png">
